app/services/deploy_status_checker.rb

```diff
@@ -1,22 +1,39 @@
 # Loads data from the /api/deploy endpoint in our deploys of identity-idp
 class DeployStatusChecker
-  Deploy = Struct.new(:env, :host)
+  Deploy = Struct.new(:app, :env, :host)
+
+  Environment = Struct.new(:env, :deploys)
 
   DEPLOYS = [
-    Deploy.new('prod'),
-    Deploy.new('demo', 'idp.demo.login.gov'),
-    Deploy.new('dev', 'idp.dev.login.gov'),
-    Deploy.new('qa', 'idp.qa.login.gov')
+    Deploy.new('identity-idp', 'prod'),
```

Comment:
note to future selves, we probably want to start pulling these from a config or something so we don't have to hardcode in the source code